src/Components/InfoContainer/VariableMapContainer/VariableMapContainer.js

```
@@ -0,0 +1,29 @@
+import React, {useEffect, useRef} from "react";
+
+import "./VariableMapContainer.scss";
+
+/**
+ * @return {JSX.Element}
+ */
+export function VariableMapContainer ({}) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)